examples/vdb-queries/src/main.py

```diff
@@ -85,13 +84,15 @@ async def on_start(self):
         await asyncio.sleep(5)
         await vehicle.Speed.subscribe(on_speed_update)
         (
-            await vehicle.Cabin.Seat.element_at(1, 1)
+            await vehicle.Cabin.Seat.Row(1)
+            .Pos(1)
```

Comment:
I would prefer one line (but don't really care):
```suggestion
            await vehicle.Cabin.Seat.Row(1).Pos(1)
```

Reply:
I agree :). However, this was split by the pre-commit which I can't change back